modules/apple_support/0.11.0/presubmit.yml

```diff
@@ -0,0 +1,20 @@
+matrix:
+  platform: ["centos7", "debian10", "macos", "ubuntu2004", "windows"]
+
+tasks:
+  verify_targets:
+    name: "Build targets under //lib"
+    platform: ${{ platform }}
+    build_targets:
+    - '@apple_support//lib/...'
+
+bcr_test_module:
+  module_path: "test"
```

Comment:
still surprised by this, and I assume this would break things in more complex repos wiht inter dependencies on absolute labels outside of the tests directory? 🤔

Reply:
You are right, this isn't working correctly right now. Because the "test" directory needs to have a WORKSPACE file and work as a separate Bazel project which depends on the target module. I replied more here: https://github.com/bazelbuild/apple_support/pull/100#discussion_r787610126